examples/dreambooth/train_dreambooth.py

```diff
@@ -184,6 +190,12 @@ def parse_args(input_args=None):
             " *output_dir/runs/**CURRENT_DATETIME_HOSTNAME***."
         ),
     )
+    parser.add_argument(
```

Comment:
This is not really related to the PR no? Could we maybe make this a seperate PR?